### DISCUSSION TITLE
Interaction

### DISCUSSION THREAD

**Discussion by hes91** [IBAX-io/go-ibax#1149] [Ideas] [answered]
### Clear and concise description of the problem

Rewarding the people who show the most interaction and support the IBAX project on social media.

### Suggested solution

A certain amount of IBXC coin pool should be created, the people who support the project and have the most interaction should be rewarded according to percentages and distributed to 100 people by drawing.
1-5 250 IBXC coins
6-25 150 IBXC Coins
26-50 100 IBXC Coins
51-75 50 IBXC Coins
76-100 25 IBXC Coins
If such a system is created and people who believe and support the project receive extra awards, it may become more attractive to people.

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by ibaxio** [comment, marked as answer]
thanks for your advice, and our community grogram will coming after the airdrop.
